src/main/java/com/google/codeu/servlets/FormHandlerServlet.java

```diff
@@ -168,21 +172,34 @@ private BlobKey getBlobKey(HttpServletRequest request, String formInputElementNa
     }
 
     /**
-     * Creates a UserMeal from the form entered by the user and then stores that meal into Datastore.
-     * TODO adjust so that form can take multiple foods at once to create the entire meal and then iterate to add all foods to meal
+     * Creates a EatenMeal from the form entered by the user and then stores that meal into Datastore.
      *
      * @param request
      * @param blobKey
      * @return meal that was stored
      */
-    private UserMeal addMealForUser(HttpServletRequest request, BlobKey blobKey) {
+    private EatenMeal addMealForUser(HttpServletRequest request, BlobKey blobKey) throws HttpStatusException {
         Datastore datastore = new Datastore();
         // Get the message entered by the user.
         String foodName = request.getParameter("foodName");
         double amount = Double.parseDouble(request.getParameter("amount"));
         String date = request.getParameter("date");
+        String mealTag = request.getParameter("mealType");
         String imageUrl;
 
+        MealType mealType;
+        if (mealTag.equals("breakfast")) {
```

Comment:
Probably need to handle lower-case vs upper- case? So when the user enters "Breakfast", it can be converted to "breakfast"

Reply:
the user doesn't type anything in. they choose a radio box, whose value is a the lowercase version of all the meal types